girder/api/docs.py

```diff
@@ -112,13 +112,25 @@ def removeRouteDocs(resource, route, method, info, handler):
                 del routes[resource]
 
 
-def addModel(name, model):
+def addModel(resource, name, model):
     """
-    This is called to add a model to the swagger documentation.
+    Add a model to the Swagger documentation.
 
+    :param resource: The type of existing resource to add the model to.
+    :param resource: str
     :param name: The name of the model.
     :type name: str
     :param model: The model to add.
     :type model: dict
```

Comment:
I don't have a problem making the changes if @cpatrick is OK with the change. I think a convenience method might be nice and if you have may ends point you are going to have to make a lot of calls. I am also working on added a test case so no one will try to remove this in the future :smile_cat:, I will push to the tip of your branch.